src/main/java/org/killbill/billing/payment/api/PaymentApi.java

```diff
@@ -365,7 +365,7 @@ public Payment createCreditWithPaymentControl(Account account, UUID paymentMetho
      * @return the payment
      * @throws PaymentApiException
      */
-    public Payment getPayment(UUID paymentId, boolean withPluginInfo, Iterable<PluginProperty> properties, TenantContext context)
+    public Payment getPayment(UUID paymentId, boolean withPluginInfo, boolean withAttempts, Iterable<PluginProperty> properties, TenantContext context)
```

Comment:
@pierre did you mean updating these methods in the PaymentApi?:
- [getAccountPayments](https://github.com/javier-gomez-hs/killbill-api/blob/fde6c74dbf9e0ba2b9ea11d5f8ec8e7b90d62e73/src/main/java/org/killbill/billing/payment/api/PaymentApi.java#L357-L357)
- [getPaymentByExternalKey](https://github.com/javier-gomez-hs/killbill-api/blob/fde6c74dbf9e0ba2b9ea11d5f8ec8e7b90d62e73/src/main/java/org/killbill/billing/payment/api/PaymentApi.java#L379-L379)
- [getPayments](https://github.com/javier-gomez-hs/killbill-api/blob/fde6c74dbf9e0ba2b9ea11d5f8ec8e7b90d62e73/src/main/java/org/killbill/billing/payment/api/PaymentApi.java#L392-L392)
- [getPayments](https://github.com/javier-gomez-hs/killbill-api/blob/fde6c74dbf9e0ba2b9ea11d5f8ec8e7b90d62e73/src/main/java/org/killbill/billing/payment/api/PaymentApi.java#L406-L406)
- [searchPayments](https://github.com/javier-gomez-hs/killbill-api/blob/fde6c74dbf9e0ba2b9ea11d5f8ec8e7b90d62e73/src/main/java/org/killbill/billing/payment/api/PaymentApi.java#L422)
- [searchPayments](https://github.com/javier-gomez-hs/killbill-api/blob/fde6c74dbf9e0ba2b9ea11d5f8ec8e7b90d62e73/src/main/java/org/killbill/billing/payment/api/PaymentApi.java#L440)